packages/protocol/contracts-0.8/common/EpochManager.sol

```diff
@@ -218,16 +222,9 @@ contract EpochManager is
   }
```

Comment:
Few lines higher we use 
`getElection().distributeEpochRewards()` - this has modifier of onlyVm in Election which needs to be updated to allow also EpochManager to distribute